### ISSUE TITLE
Spreading attributes from object requires comma

### ISSUE THREAD

**Issue by ezhlobo** [pugjs/babel-plugin-transform-react-pug#101]
Better to show an example:

### Invalid
```pug
button(type="button" ...props)
```

[Proof](https://react.pugjs.org/repl#%7B%22code%22:%22ZXhwb3J0IGNvbnN0IFJlYWN0Q29tcG9uZW50ID0gcHJvcHMgPT4gcHVnYAogIGJ1dHRvbih0eXBlPSJidXR0b24iIC4uLnByb3BzKQpg%22%7D)

### Valid
```pug
button(type="button", ...props)
```

[Proof](https://react.pugjs.org/repl#%7B%22code%22:%22ZXhwb3J0IGNvbnN0IFJlYWN0Q29tcG9uZW50ID0gcHJvcHMgPT4gcHVnYAogIGJ1dHRvbih0eXBlPSJidXR0b24iLCAuLi5wcm9wcykKYA==%22%7D)

Will be nice to have both cases working.

**Comment by ForbesLindesay** [comment]
This should just be a case of treating `...` as a special case like we do with `:` in https://github.com/pugjs/pug/blob/2f1110ec2ed8656253331e1e4b1b20b9d12185ee/packages/pug-lexer/index.js#L1221

**Comment by ezhlobo** [comment]
@ForbesLindesay thank you, that helped! Created pull request to pug-lexer:

https://github.com/pugjs/pug/pull/3152

**Comment by ezhlobo** [comment]
Fixed in [v7.0.1](https://github.com/pugjs/babel-plugin-transform-react-pug/releases/tag/v7.0.1)
